tools/build/make_markdown

```diff
@@ -0,0 +1,114 @@
+#! /usr/bin/env bash
+
+awk '
+function output(level, old_level, label, basic, intermediate, advanced, text) {
+	#bad_char = "🤢";
+	#bad_char = "❌";
+	bad_char = "?";
+	#check_char = "✓"
+	check_char = "✔️";
+	emdash = "—";
+
+	if (level == 1) {
+		if (label == "?") {
+			label = bad_char;
+			#basic = emdash;
+			#intermediate = emdash;
+			#advanced = emdash;
+		} else {
+			if (basic == "y") {
+				#basic = sprintf("<a href=\"#%s-basic\">%s</a>", label, check_char);
```

Comment:
You want to keep these commented out lines here?

Reply:
Fixed